.gitignore

```diff
@@ -0,0 +1,4 @@
+*.log
+*.pot
+*.pyc
+__pycache__
```

Comment:
File này lỗi indent nè, thêm line trống cuối file nha. Em setting trong vs code để nó auto chèn line trống nhé
https://stackoverflow.com/questions/68884005/what-is-the-shortcut-to-auto-indent-code-in-vscode